sklearn/model_selection/_search.py

```diff
@@ -589,9 +589,9 @@ def fit(self, X, y=None, groups=None, **fit_params):
             Training vector, where n_samples is the number of samples and
             n_features is the number of features.
 
-        y : array-like, shape = [n_samples] or [n_samples, n_output], optional
-            Target relative to X for classification or regression;
-            None for unsupervised learning.
+        y : {array-like, sparse matrix} shape = [n_samples] or
+            [n_samples, n_output], optional. Target relative to X for
```

Comment:
Same comment about line ending: the line should go on up to "optional" and end there (you can protect the line return, to avoid having too long a line).